akita/src/devtools.ts

```diff
@@ -104,6 +105,21 @@ export function akitaDevtools(ngZoneOrOptions?: NgZoneLike | Partial<DevtoolsOpt
         console.groupEnd();
       }
 
+      if (options.sortAlphabetically) {
+        const sortedAppState = {};
```

Comment:
It can be shorter :)

```ts
 const sorted = Object.keys(appState)
    .sort()
    .reduce((acc, storeName) => {
      acc[storeName] = appState[storeName];
      return acc;
    }, {});
```

Reply:
totally agree! :)